pymc/distributions/multivariate.py

```diff
@@ -1827,16 +1827,16 @@ class MatrixNormal(Continuous):
         with pm.Model() as model:
             # Setup right cholesky matrix
             sd_dist = pm.HalfCauchy.dist(beta=2.5, shape=3)
-            colchol_packed = pm.LKJCholeskyCov('colcholpacked', n=3, eta=2,
-                                               sd_dist=sd_dist)
+            colchol_packed = pm.LKJCholeskyCov('colcholpacked', n=3, eta=2,compute_corr=False,
+                                            sd_dist=sd_dist)
```

Comment:
A bit more succinct?
```suggestion
            colchol_packed, _, _ = pm.LKJCholeskyCov('colcholpacked', n=3, eta=2, sd_dist=sd_dist)
```

Reply:
@ricardoV94  Yeah also if we are using this approach then there is No need for expand_packed_triangular(line:1832) since it's already expanded. Also I will rename "cholchol_packed" to "cholchol"